README.md

```diff
@@ -22,7 +22,7 @@ This docker-compose includes the Celery worker that was introduced with Nautobot
 
```

Comment:
```suggestion
This documentation is now written assuming the latest Docker Compose methodology of using the Docker Compose Plugin instead of the independent docker-compose executable. See the [Docker Compose Plugin installation notes](https://docs.docker.com/compose/install/)
```